src/plots/plots.js

```diff
@@ -809,6 +809,16 @@ plots.purge = function(gd) {
     // remove modebar
     if(fullLayout._modeBar) fullLayout._modeBar.destroy();
 
+    if(fullLayout._plots) {
```

Comment:
Hmm. Gl context deletion should be already taken care of via [`Plots.cleanPlot`](https://github.com/plotly/plotly.js/blob/46bbbaebfd70ed28196df9c4993a1ded5292bebf/src/plots/plots.js#L559) which is currently called in `Plots.supplyDefaults` [here](https://github.com/plotly/plotly.js/blob/46bbbaebfd70ed28196df9c4993a1ded5292bebf/src/plots/plots.js#L520) and more explicity in `Plotly.purge` [here](https://github.com/plotly/plotly.js/blob/46bbbaebfd70ed28196df9c4993a1ded5292bebf/src/plot_api/plot_api.js#L2492).

Maybe something is up with the gl2d `clean` [method](https://github.com/plotly/plotly.js/blob/46bbbaebfd70ed28196df9c4993a1ded5292bebf/src/plots/gl2d/index.js#L68-L85)?